luma/core/multiimage.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+from PIL import Image, ImageDraw 
+from luma.core import mixin 
+
+
+class MultiImage(mixin.capabilities):
+    '''
+    Renders multiple images on a background
+    '''
+
+    image_id = 0
+
+    def __init__(self, device, width, height):
+        '''
+        Instantiates a new MultiImage
+        '''
+        self.capabilities(width, height, rotate=0, mode=device.mode)
+        self._device = device
+        self._background_image = Image.new(self.mode, self.size)
+        self.images = {}
```

Comment:
Using an unordered dict here - when iterating over them and pasting into the background image is done in an unordered way, and may change as new images are added.

This could be important if images overlap?

Reply:
Changed it to a normal list